samples/document-snippets/cluster.js

```diff
@@ -129,6 +129,86 @@ const snippets = {
       });
     // [END bigtable_cluster_set_meta]
   },
+
+  createBackup: (instanceId, clusterId, tableId, backupId) => {
```

Comment:
These snippets are tech debt :/  I believe we don't have tests for many of them.  For any new sample code, please create a new file, and follow https://github.com/googleapis/nodejs-bigtable/blob/master/samples/quickstart.js as a guide.  One per file!

Reply:
Thanks for reviewing the samples. There were competing patterns, and I wasn't sure what to implement. I'll revise based on your feedback.